### ISSUE TITLE
Rationale?

### ISSUE THREAD

**Issue by drom** [parallella/openeda#2]
@aolofsson besides intriguing everybody, could you give us a bit of rationale/abstract behind this repo?
:confused:

**Comment by aolofsson** [comment]
Still very much work in progress....putting all the work I can out in the open so that others can benefit, although I realize it's mostly snippets right now. Chip design is an ugly business.

**Comment by drom** [comment]
Yes, you are right it is an ugly business. How about "Making Chip design a better place" as a mission statement?
